tools/cmake/open62541Config.cmake.in

```diff
@@ -8,8 +8,6 @@ set (open62541_NODESET_DIR @PACKAGE_open62541_install_nodeset_dir@ CACHE PATH "P
 include(CMakeFindDependencyMacro)
 find_dependency(PythonInterp REQUIRED)
 
-include("${CMAKE_CURRENT_LIST_DIR}/open62541Macros.cmake")
```

Comment:
Should the open62541Macros.cmake be removed from the main CMakeLists.txt as well?

Reply:
Nope, this patch should be removed, the file is actually there in this branch.